packages/database/src/__tests__/modelClasses/Task.test.js

```diff
@@ -100,27 +100,29 @@ describe('TaskModel', () => {
 
     await Promise.all(SURVEYS.map(survey => upsertDummyRecord(models.survey, survey)));
 
+    const dueDate = new Date('2020-01-01 00:00:00+00').getTime();
```

Comment:
Wouldn't it just be easier if we didn't have all these tests to worry about and maintain ;)